pkgs/development/libraries/nyan/default.nix

```diff
@@ -0,0 +1,40 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, clang_14
+, cmake
+, flex
+}:
+
+stdenv.mkDerivation {
+  pname = "nyan";
+  version = "0.2.0";
+
+  src = fetchFromGitHub {
+    owner = "SFTtech";
+    repo = "nyan";
+    rev = "7b5517f8c3c17b1fb2483f6dec882ccb84b70bbb";
```

Comment:
```suggestion
    rev = "v${version}";
```

Or maybe 

```suggestion
    rev = "refs/tags/${version}"; # because pure ${version} doesn't work
```

Reply:
It won't compile if I do this, so I think we should leave it out

Reply:
This is unusual. Can you show the error log?

Reply:
I don't know why changing this would change this but I think this is beyond me
(Not sure how to show the error log so going to put it directly)

> trying https://github.com/SFTtech/nyan/archive/refs/tags/0.2.tar.gz
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0    14    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
curl: (22) The requested URL returned error: 404
error: cannot download source from any mirror
error: builder for '/nix/store/qjiy2g0l0yiizn2h618pifqbbs7smpab-source.drv' failed with exit code 1;
       last 8 log lines:
       >
       > trying https://github.com/SFTtech/nyan/archive/refs/tags/0.2.tar.gz
       >   % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
       >                                  Dload  Upload   Total   Spent    Left  Speed
       >   0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
       >   0    14    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
       > curl: (22) The requested URL returned error: 404
       > error: cannot download source from any mirror
       For full logs, run 'nix log /nix/store/qjiy2g0l0yiizn2h618pifqbbs7smpab-source.drv'.
error: 1 dependencies of derivation '/nix/store/vx6zwd0mw76v6pgajyvfrn101345jdjr-nyan-0.2.drv' failed to build